ci-operator/jobs/openshift/installer/openshift-installer-main-presubmits.yaml

```diff
@@ -2576,7 +2576,7 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )aro-unit,?($|\s.*)
   - agent: kubernetes
-    always_run: true
+    always_run: false
```

Comment:
always_run: true for artifacts-images?

Reply:
I think it is OK in this case to make always_run: false because we are skipping only for non code changes. Thanks for the careful review.